src/quorum/testdata/joint_group_commit.txt

```diff
@@ -0,0 +1,76 @@
+# the same result of joint single group commit
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(1,1,1)
+----
+100
+
+# min(quorum_commit_index = 100, first index that appears in second group = 99)
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(1,1,2)
+----
+99
+
+# min(quorum_commit_index = 100, first index that appears in second group = 101)
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(2,1,1)
+----
+100
+
+# minimum index = 99
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(_,1,1)
+----
+99
+
+# min(quorum_commit_index = 100, first index that appears in second group = 99)
+group_committed cfg=(1,2,3) cfgj=zero idx=(100,101,99) gid=(_,1,2)
+----
+99
+
+# min(quorum_commit_index = 100, first index that appears in second group = 98)
```

Comment:
Actually it should be minimum commit index because there is just one known group.